backend/repository/config.go

```diff
@@ -117,8 +122,13 @@ type ThriftMeta struct {
 type ProtocolType string
 
 const (
-	gatewayConfigFile = "gateway.json"
+	gatewayConfigFile     = "gateway.json"
+	productionCfgJSONPath = "config/production.json"
+	clientConfigFileName  = "client-config.json"
+	clientModuleFileName  = "clients-config.json"
```

Comment:
`clientsConfigFileName` is arguably a better name from module system point of view, despite it's awfully close to `clientConifgFileName`